package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Libraries to ease pouch-couch stack on offline first app",
   "main": "index.js",
   "scripts": {
+    "test:eslint": "eslint --fix ./",
```

Comment:
ini ganti aja `"lint": "eslint --fix ./",`

Reply:
reject this PR